### ISSUE TITLE
How to use Arkenfox on the Flatpak version of Firefox [A: Flatpak uses a different path for profiles]

### ISSUE THREAD

**Issue by Bilborne** [arkenfox/user.js#1713]
I have tried a plethora of ways to get Arkenfox working with the Flatpak version of Firefox to no avail. 
I did manage to get it working by adding the following via Flatseal: /home/.mozzilla/
It also worked with the following as well: /home/.mozzilla/firefox/
However I do not like the above mentioned approach because it uses the native Firefox directory.
Any tips would be welcomed;

Thank you.

**Comment by Svallinn** [comment]
Flatpak app data is on `~/.var/app/<app_name>`
For Firefox, `~/.var/app/org.mozilla.firefox/.mozilla` is the Flatpak equivalent to the more native `~/.mozilla`

**Comment by Bilborne** [comment]
I have figured out a solution for using Arkenfox with the Flatpak version of Firefox.
Here's what I did:

1. I went to ~/.var/app/org.mozilla.firefox/.mozilla/firefox/PROFILE, and put in the prefsCleaner.sh, Updater.sh, and user.js files.(I did not import my user-overrides.js file because I was only testing this)

2. I ran updater.sh and prefsCleaner.sh respectively.

3. I launched Firefox( Flatpak version), and I was using Arkenfox user.js!!

The reason why I was not able to use Arkenfox with the Flatpak version of Firefox before was because I cannot access the root directory in the "about:profiles" page.
However the root directory can be found by going directly into ~/.var/app/org.mozilla.firefox/.mozilla/firefox/PROFILE

Thank you for your help!
